generators/app/templates/src/app/core/shell/_shell.component.scss

```diff
@@ -1,4 +1,38 @@
 <% if (props.ui === 'ionic') { -%>
+<%   if (props.layout === 'tabs') { -%>
+$tabs-height: 8.2rem;
+
+.tabs::ng-deep {
+  .content {
+    .scroll-content,
+    .fixed-content {
+      height: 100%;
+    }
+  }
+
+  @media (min-width: 768px) {
```

Comment:
use $screen-breakpoints /md variable for this, and it may even have a mixin for the media query